### ISSUE TITLE
TODO: document how to re-generate the signatures if you do not have the agent private key, but you do have the repository private key

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaCI/julia-buildkite#110]
In #103, we added basic documentation on how a maintainer can re-generate all of the signatures in this repository if they have the agent private key file.

But we should also have the documentation for the (more common) use case where you only have the repository private key file, and you want to re-generate all signatures.

**Comment by thazhemadam** [comment]
Isn't it as straightforward as this or am I missing something?
 
```bash
git clone git@github.com:JuliaCI/julia-buildkite.git
cd julia-buildkite
git checkout YOURINITIALS/YOUR-BRANCH-NAME
git clone https://github.com/staticfloat/cryptic-buildkite-plugin.git
mv /path/to/my/repository/private/key ./cryptic_repo_keys/repo_key
make sign_treehashes
git push origin YOURINITIALS/YOUR-BRANCH-NAME
```

If this looks right, I can open a PR. 🙂

**Comment by DilumAluthge** [comment]
I checked with Elliot, and he confirmed that your comment is correct!

@thazhemadam It would be great if you could make a PR.

**Comment by thazhemadam** [comment]
@DilumAluthge done! https://github.com/JuliaCI/julia-buildkite/pull/284

**Comment by DilumAluthge** [comment]
Fixed by #284
